packages/xrpc-server/src/stream/server.ts

```diff
@@ -9,38 +10,53 @@ export class XrpcStreamServer {
   constructor(opts: ServerOptions & { handler: Handler }) {
     const { handler, ...serverOpts } = opts
     this.wss = new WebSocketServer(serverOpts)
-    this.wss.on('connection', async (socket, req) => {
-      socket.on('error', (err) => logger.error(err, 'websocket error'))
+    this.wss.on('connection', async (ws, req) => {
```

Comment:
Renamed to `ws` to avoid confusion with the internal `_socket` property (`socket._socket` feels wierd)